docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md

```diff
@@ -0,0 +1,228 @@
+---
+id: router-configuration
+title: Router Configuration
+sidebar_label: Configuration
+hide_table_of_contents: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This article expands on the [Linux and Docker router deployment](/guides/deployments/10-linux/20-router/10-deploy.mdx) article with configuration concepts and examples. [Kubernetes is covered separately](/guides/deployments/30-kubernetes//kubernetes-router.mdx).
```

Comment:
This whole paragraph should be removed imo and this doc moved into reference somewhere. it doesn't feel guide related at all to me and shouldn't call out linux, nor kubernetes etc.